app/screens/planner/planner-screen.tsx

```diff
@@ -92,6 +95,13 @@ const CHANGE_DIRECTION_WRAPPER: ViewStyle = {
   zIndex: 10,
 }
 
+const RESET_TIME_ICON_STYLE: ImageStyle = {
+  height: 24,
+  width: 24,
+  resizeMode: "contain",
+  tintColor: "black",
```

Comment:
Icon color is black in dark mode
```suggestion
  tintColor: color.text,
```